libs/application/template-api-modules/src/lib/modules/templates/secondary-school/emailGenerators/applicationSubmittedEmail.ts

```diff
@@ -0,0 +1,80 @@
+import { SecondarySchoolAnswers } from '@island.is/application/templates/secondary-school'
+import { Message } from '@island.is/email-service'
+import { EmailTemplateGeneratorProps } from '../../../../types'
+import { EmailRecipient } from '../types'
+import { pathToAsset } from '../utils'
+import { ApplicationConfigurations } from '@island.is/application/types'
+
+export type ApplicationSubmittedEmail = (
+  props: EmailTemplateGeneratorProps,
+  recipient: EmailRecipient,
+) => Message
+
+export const generateApplicationSubmittedEmail: ApplicationSubmittedEmail = (
+  props,
+  recipient,
+): Message => {
+  const {
+    application,
+    options: { email, clientLocationOrigin },
+  } = props
+  const answers = application.answers as SecondarySchoolAnswers
```

Comment:
Yes, excellent point. My main issue is the use of the "as" keyword since it is tricking the linter and the answers are not handed type safe to the application-template-loaders